profiles/models.py

```diff
@@ -7,6 +7,23 @@
 from jsonfield import JSONField
 
 
+class Employment(models.Model):
+    """
+    A user work_history
+    """
+    city = models.TextField(blank=True, null=True)
+    company_name = models.TextField(blank=True, null=True)
+    country = models.TextField(blank=True, null=True)
+    industry = models.TextField(blank=True, null=True)
+    position = models.TextField(blank=True, null=True)
+    end_date = models.DateTimeField(blank=True, null=True, auto_now_add=True)
```

Comment:
We don't want `auto_now_add` here or below, we cannot fill in the employment date for the user, they have to provide this information